player/Audio_Scope.h

```diff
@@ -3,9 +3,9 @@
 #ifndef AUDIO_SCOPE_H
 #define AUDIO_SCOPE_H
 
-#include "SDL.h"
 #include "gme/gme.h"
-
+#include <SDL_render.h>
```

Comment:
Recommended include method for SDL2 headers is quotes instead of brackets: replace `<SDL_xxx.h>` with `"SDL_xxx.h"`